### DISCUSSION TITLE
part-ii/2021-87/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#345] [Question comments]
Part II 2021 Logic and Set Theory: Paper 1, Section II, 16G 

https://questions.tripos.org/part-ii/2021-87/

**Comment by artie2000** [comment]
(a) is bookwork.

<details>
<summary>(b)</summary>

Let $F=\{f_1,\dots,f_n\}$, and let $f=\bigwedge_{i=1}^n f_i$; then $S$ is equivalent to $f$. As in (a), some finite subset $R\subseteq S$ proves $f$, so $R\vdash S$.

</details>

<details>
<summary>(c)</summary>

Suppose $T_i\vdash\bot$. Then $T_i$ proves all propositions, so $T_i=L$ as is it deductively closed. But $T_{i+1}\supsetneq T_i$.$_\#$

</details>

<details>
<summary>Rider</summary>

The (finitely many) hypotheses in any proof of from $T$ are contained in some $T_i$.

Therefore $T\vdash\bot$ implies that $T_i\vdash\bot$,$_\#$ and $T\vdash p$ implies that $p\in T_i\subseteq T$.

If some finite $F\vdash T$, then some $T_i\vdash T$, so $T=T_i$. But $T_{i+1}\supsetneq T_i$.$_\#$

</details>
